build-tools-internal/src/main/java/org/elasticsearch/gradle/internal/ElasticsearchJavaBasePlugin.java

```diff
@@ -165,6 +172,25 @@ public static void configureInputNormalization(Project project) {
         project.getNormalization().getRuntimeClasspath().ignore("IMPL-JARS/**/META-INF/MANIFEST.MF");
     }
 
+    private static void configureNativeLibraryPath(Project project) {
+        String nativeProject = ":libs:elasticsearch-native:libraries";
```

Comment:
Just a heads up that we will need to make some changes to the serverless build to replace this dependency.